src/gc/gc.cpp

```diff
@@ -10442,7 +10442,7 @@ gc_heap::init_gc_heap (int  h_number)
     make_background_mark_stack (b_arr);
 #endif //BACKGROUND_GC
 
-    adjust_ephemeral_limits();
+    adjust_ephemeral_limits(true);
```

Comment:
Yea I usually do that in new code. The change has already been merged, so I'll leave it for now.